staging/src/k8s.io/client-go/rest/transport.go

```diff
@@ -118,3 +128,45 @@ func (c *Config) TransportConfig() (*transport.Config, error) {
 func (c *Config) Wrap(fn transport.WrapperFunc) {
 	c.WrapTransport = transport.Wrappers(c.WrapTransport, fn)
 }
+
+var CertificateCacheExpiryPeriod = time.Second
+
+type certificateCacheEntry struct {
+	*tls.Certificate
+	error
+	time.Time
+}
+
+func (c *certificateCacheEntry) isExpired() bool {
```

Comment:
ordinarily this name is fine, but expiry on certs means something different.  Doc it.